module/preference/PreferenceDialog.kt

```diff
@@ -350,7 +354,7 @@ object PreferenceDialog {
                 }
             }
             SpacedHorizontalSeparator()
-            preferences.forEach { it.Display() }
+            preferences.forEach { it.Display(); ColumnSpacer() }
```

Comment:
We didn't have multiple preference fields in a single group before - it looks much better with some spacing.